### DISCUSSION TITLE
How to set tire pressure units from bar to psi at default on 1.27.0?

### DISCUSSION THREAD

**Discussion by ktsenvolt** [teslamate-org/teslamate#2769] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/80360674/179405400-70e7eea5-07d8-4e03-8af8-b08492b24be6.png)
How to set tire pressure units from bar to psi at default on 1.27.0?
Thanks.

**Comment by LelandSindt** [comment, marked as answer]
At this time there is no way to set a default, a change is in the works.... https://github.com/adriankumpf/teslamate/pull/2727
